waku/v2/node/waku_setup.nim

```diff
@@ -0,0 +1,115 @@
+{.push raises: [Defect].}
```

Comment:
I'd suggest using a different name like `waku_setup_utils` or `waku_setup_helpers`  to indicate this module contains the helpers, not the setup functions